flexmeasures/cli/db_ops.py

```diff
@@ -136,4 +146,15 @@ def restore(file: str):
         click.secho("db restore unsuccessful", **MsgStyle.ERROR)
 
 
+def pg_cluster_arg_from_version(pg_version: str) -> str:
+    """
+    Small utility, turning the target postgres version into an arg for pg utils,
+    so that they know what they're supposed to work against.
+    More info (e.g. on the format) at https://manpages.ubuntu.com/manpages/trusty/man1/pg_wrapper.1.html
+    """
+    if pg_version:
+        return f"--cluster {pg_version}/main"
```

Comment:
This didn't work for me. I got an `Error: Invalid version 15 specified with --cluster`. However, changing this line to `return f"--version {pg_version}"` made the dump work. I'm running PostgreSQL 13.5. I'm guessing you are running version 14, so perhaps you can test whether my remedy would suit you, too?

Reply:
I just implemented this change in my last commit for your testing convenience. If it works for you, too, please consider refactoring (e.g. renaming) `pg_cluster_arg_from_version`.